osu.Game/Overlays/Settings/SettingsItem.cs

```diff
@@ -100,9 +100,23 @@ public virtual Bindable<T> Current
 
         public IEnumerable<string> Keywords { get; set; }
 
-        public override bool IsPresent => base.IsPresent && MatchingFilter;
+        private bool matchingFilter;
```

Comment:
This used to be set to true by default. It isn't now, which is probably causing the test failures. Same thing in `SettingsSection` too.